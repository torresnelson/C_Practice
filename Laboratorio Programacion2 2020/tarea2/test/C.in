# Caso de prueva privado 3 
estaOrdenadaPorNaturales
insertarSegmentoDespues 8 (1,3.4) (2,4.1) (3,3.5) (4,9.4) (5,2.6) (5,3.5) (7,9.4) (7,2.6)
estaOrdenadaPorNaturales
borrarSegmento 1 8
estaOrdenadaPorNaturales
insertarSegmentoDespues 8 (2,3.4) (2,4.1) (1,3.5) (4,9.4) (5,2.6) (5,3.5) (7,9.4) (7,2.6)
estaOrdenadaPorNaturales
finalCadena
insertarSegmentoDespues 4 (7,9.4) (7,2.6) (7,9.4) (7,2.6)
estaOrdenadaPorNaturales
borrarSegmento 3 3 
estaOrdenadaPorNaturales
imprimirCadena
Fin
