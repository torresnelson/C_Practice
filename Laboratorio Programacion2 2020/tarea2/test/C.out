1>#  Caso de prueva privado 3 .
2>cad ordenada.
3>Segmento insertado después de loc.
4>cad ordenada.
5>Segmento borrado.
6>cad ordenada.
7>Segmento insertado después de loc.
8>cad no ordenada.
9>loc al final.
10>Segmento insertado después de loc.
11>cad no ordenada.
12>Segmento borrado.
13>cad ordenada.
14>(2,3.40)(2,4.10)(4,9.40)(5,2.60)(5,3.50)(7,9.40)(7,2.60)(7,9.40)(7,2.60)(7,9.40)(7,2.60)
15>Fin.
